recipe/build.sh

```diff
@@ -1,19 +1,23 @@
+#!/usr/bin/env bash
+set -eux
+
+sed -i -e \
+    "s;^#define LIBDIR ;#define LIBDIR \"${PREFIX}/share/hunspell:${PREFIX}/share/hunspell_dictionaries:\" ;" \
+    src/tools/hunspell.cxx
+
 autoreconf -vfi
-./configure --prefix=$PREFIX --with-readline --with-ui
-make
-if [[ "${CONDA_BUILD_CROSS_COMPILATION:-}" != "1" || "${CROSSCOMPILING_EMULATOR}" != "" ]]; then
-make check
+
+./configure \
+    --prefix="${PREFIX}" \
+    --with-readline \
+    --with-ui
+
+make "-j${CPU_COUNT}"
+
+if [[ "${target_platform}" == "${build_platform}" ]]; then
```

Comment:
The CONDA_BUILD_CROSS_COMPILATION and CROSSCOMPILING_EMULATOR are from the cross-compile example here: https://conda-forge.org/docs/maintainer/knowledge_base/#cross-compilation-examples

If I understand, CONDA_BUILD_CROSS_COMPILATION != 1 is equivalent to the target_platform == build_platform check. And this would remove support for using an emulator.

Reply:
I think `linux-aarch64` (and certainly  `-ppc64le`) _are_ emulated on Azure, so reckon it's already working...

Reply:
(and one of them was in fact undefined on one of the pushes osx... would have to dig through the logs)

Reply:
Because you set the build platform to `linux-aarch64: linux-64`, this should be doing cross-compilation. The CROSSCOMPILING_EMULATOR is for support for running target binaries on the build platform.

Reply:
I verified in one of the previous Azure build logs, for aarch this check is `+ [[ linux-aarch64 == \l\i\n\u\x\-\6\4 ]]`. So it will skip `make check` on the aarch64/ppc builds.

Normally we could still run `make check` as CROSSCOMPILING_EMULATOR would indicate that the x86 build platform supports running aarch64 binaries. However this may not work anyway, as we install some extra gconv libraries, and those would not be present in the cross-compiled toolchain.

Reply:
Yeah: the tests (and, I guess, the manpages) are pretty sensitive to locale...